### PR TITLE
Update SLE-HA Guide to sentence case (WIP)

### PR DESCRIPTION
**Pull request by sknorr**
### Description

* The new capitalization rules are documented at https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-capitalization, read carefully
* This is the result of an automatic conversion and will most likely need further adjustments
  * Make sure to review the entire diff
  * Entity files have not been changed and need to be updated manually
  * Some product/project names may have been inadvertently changed to the wrong capitalization
  * Some titles may have been missed, either because they're split across multiple source lines or because of improper markup
* Update the conversion commit and remove the (WIP) from the commit message before merging
* This is the first PR in a series of four PRs

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLEHA12SP4
- [ ] To maintenance/SLEHA15
- [ ] To maintenance/SLEHA15SP1

### CODE REVIEW

File: xml/ha_drbd.xml [hunk 1]
```diff
@@ -483,7 +483,7 @@ NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE
 
     <variablelist>
       <varlistentry>
-        <term><guimenu>Resource Name</guimenu></term>
+        <term><guimenu>Resource name</guimenu></term>
```
Comment (taroth21) [inline xml/ha_drbd.xml:486]:
```suggestion
        <term><guimenu>Resource Name</guimenu></term>
```

File: xml/ha_drbd.xml [hunk 2]
```diff
@@ -495,7 +495,7 @@ NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE
         </listitem>
       </varlistentry>
       <varlistentry>
-        <term><guimenu>Address:Port</guimenu></term>
+        <term><guimenu>Address:port</guimenu></term>
```
Comment (taroth21) [inline xml/ha_drbd.xml:498]:
```suggestion
        <term><guimenu>Address:Port</guimenu></term>
```

File: xml/ha_fencing.xml [hunk 3]
```diff
@@ -57,7 +57,7 @@
 
   <variablelist>
    <varlistentry>
-    <term>Resource Level Fencing</term>
+    <term>Resource Level fencing</term>
```
Comment (taroth21) [inline xml/ha_fencing.xml:60]:
```suggestion
    <term>Resource level fencing</term>
```

File: xml/ha_fencing.xml [hunk 4]
```diff
@@ -68,7 +68,7 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Node Level Fencing</term>
+    <term>Node Level fencing</term>
```
Comment (taroth21) [inline xml/ha_fencing.xml:71]:
```suggestion
    <term>Node level fencing</term>
```

File: xml/ha_fencing.xml [hunk 5]
```diff
@@ -80,7 +80,7 @@
   </variablelist>
  </sect1>
  <sect1 xml:id="sec-ha-fencing-nodes">
-  <title>Node Level Fencing</title>
+  <title>Node Level fencing</title>
```
Comment (taroth21) [inline xml/ha_fencing.xml:83]:
```suggestion
  <title>Node level fencing</title>
```

File: xml/ha_fencing.xml [hunk 6]
```diff
@@ -109,7 +109,7 @@
    </para>
    <variablelist>
     <varlistentry>
-     <term>Power Distribution Units (PDU)</term>
+     <term>Power distribution units (PDU)</term>
```
Comment (taroth21) [inline xml/ha_fencing.xml:112]:
```suggestion
     <term>Power Distribution Units (PDU)</term>
```

File: xml/ha_fencing.xml [hunk 7]
```diff
@@ -121,7 +121,7 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Uninterruptible Power Supplies (UPS)</term>
+     <term>Uninterruptible power supplies (UPS)</term>
```
Comment (taroth21) [inline xml/ha_fencing.xml:124]:
```suggestion
     <term>Uninterruptible Power Supplies (UPS)</term>
```

File: xml/ha_migration.xml [hunk 8]
```diff
@@ -59,8 +59,8 @@
 
   <variablelist>
    <varlistentry>
-    <term>Major Release</term>
-    <term>General Availability (GA) Version</term>
+    <term>Major release</term>
+    <term>General availability (GA) version</term>
```
Comment (taroth21) [inline xml/ha_migration.xml:63]:
```suggestion
    <term>General Availability (GA) version</term>
```

File: xml/ha_migration.xml [hunk 9]
```diff
@@ -93,7 +93,7 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Service Pack (SP)</term>
+    <term>Service pack (SP)</term>
```
Comment (taroth21) [inline xml/ha_migration.xml:96]:
```suggestion
    <term>Service Pack (SP)</term>
```

File: xml/ha_rear.xml [hunk 10]
```diff
@@ -517,7 +517,7 @@
   </procedure>
 
   <example xml:id="ex-ha-rear-config-EMC">
-   <title>Using Third-Party Backup Tools Like EMC NetWorker</title>
+   <title>Using third-party backup tools like EMC netWorker</title>
```
Comment (taroth21) [inline xml/ha_rear.xml:520]:
```suggestion
   <title>Using third-party backup tools like EMC NetWorker</title>
```

File: xml/ha_requirements.xml [hunk 11]
```diff
@@ -231,19 +231,19 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Time Synchronization</term>
+    <term>Time synchronization</term>
     &sys-req-other-ntp;
    </varlistentry>
    <varlistentry>
-    <term>Network Interface Card (NIC) Names</term>
+    <term>Network interface card (NIC) names</term>
```
Comment (taroth21) [inline xml/ha_requirements.xml:238]:
```suggestion
    <term>Network Interface Card (NIC) names</term>
```

File: xml/ha_samba.xml [hunk 12]
```diff
@@ -163,7 +163,7 @@
   </para>
 
   <procedure xml:id="pro-ha-samba-basicconf">
-   <title>Setting Up a Basic Clustered Samba Server</title>
+   <title>Setting up a basic clustered samba server</title>
```
Comment (taroth21) [inline xml/ha_samba.xml:166]:
```suggestion
   <title>Setting up a basic clustered Samba server</title>
```

File: xml/ha_samba.xml [hunk 13]
```diff
@@ -372,7 +372,7 @@ Clone Set: cl-clusterfs [clusterfs]
   </procedure>
  </sect1>
  <sect1 xml:id="sec-ha-samba-ad">
-  <title>Joining an Active Directory Domain</title>
+  <title>Joining an active directory domain</title>
```
Comment (taroth21) [inline xml/ha_samba.xml:375]:
```suggestion
  <title>Joining an Active Directory domain</title>
```

File: xml/ha_samba.xml [hunk 14]
```diff
@@ -486,7 +486,7 @@ Cluster AD member NETBIOS name = "CTDB-SERVER"</screen>-->
   </para>
 
   <procedure xml:id="pro-ha-samba-config-join-ad">
-   <title>Joining Active Directory</title>
+   <title>Joining active directory</title>
```
Comment (taroth21) [inline xml/ha_samba.xml:489]:
```suggestion
   <title>Joining Active Directory</title>
```

File: xml/ha_samba.xml [hunk 15]
```diff
@@ -519,7 +519,7 @@ Cluster AD member NETBIOS name = "CTDB-SERVER"</screen>-->
   </procedure>
  </sect1>
  <sect1 xml:id="sec-ha-samba-testing">
-  <title>Debugging and Testing Clustered Samba</title>
+  <title>Debugging and testing clustered samba</title>
```
Comment (taroth21) [inline xml/ha_samba.xml:522]:
```suggestion
  <title>Debugging and testing clustered Samba</title>
```

File: xml/ha_storage_basics.xml [hunk 16]
```diff
@@ -7,7 +7,7 @@
 <!-- Converted by suse-upgrade version 1.1 -->
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-ha-storage-dlm">
- <title>Distributed Lock Manager (DLM)</title>
+ <title>Distributed lock manager (DLM)</title>
```
Comment (taroth21) [inline xml/ha_storage_basics.xml:10]:
```suggestion
 <title>Distributed Lock Manager (DLM)</title>
```

File: xml/ha_troubleshooting.xml [hunk 17]
```diff
@@ -152,7 +152,7 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>I only get a <literal>failed</literal> message. Is it possible to get more
+    <term>I only get a <literal>failed</literal> message. is it possible to get more
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:155]:
```suggestion
    <term>I only get a <literal>failed</literal> message. Is it possible to get more
```

File: xml/ha_troubleshooting.xml [hunk 18]
```diff
@@ -164,7 +164,7 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>How can I get an overview of all my nodes and resources?</term>
+    <term>How can i get an overview of all my nodes and resources?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:167]:
```suggestion
    <term>How can I get an overview of all my nodes and resources?</term>
```

File: xml/ha_troubleshooting.xml [hunk 19]
```diff
@@ -228,7 +228,7 @@ Operations:
 
   <variablelist>
    <varlistentry>
-    <term>How can I clean up my resources?</term>
+    <term>How can i clean up my resources?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:231]:
```suggestion
    <term>How can I clean up my resources?</term>
```

File: xml/ha_troubleshooting.xml [hunk 20]
```diff
@@ -243,7 +243,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>How can I list my currently known resources?</term>
+    <term>How can i list my currently known resources?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:246]:
```suggestion
    <term>How can I list my currently known resources?</term>
```

File: xml/ha_troubleshooting.xml [hunk 21]
```diff
@@ -252,7 +252,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>I configured a resource, but it always fails. Why?</term>
+    <term>I configured a resource, but it always fails. why?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:255]:
```suggestion
    <term>I configured a resource, but it always fails. Why?</term>
```

File: xml/ha_troubleshooting.xml [hunk 22]
```diff
@@ -295,7 +295,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Why can I never tell where my resource will run?</term>
+    <term>Why can i never tell where my resource will run?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:298]:
```suggestion
    <term>Why can I never tell where my resource will run?</term>
```

File: xml/ha_troubleshooting.xml [hunk 23]
```diff
@@ -340,7 +340,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Why does fencing not happen, although I have the &stonith; resource?</term>
+    <term>Why does fencing not happen, although i have the &stonith; resource?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:343]:
```suggestion
    <term>Why does fencing not happen, although I have the &stonith; resource?</term>
```

File: xml/ha_troubleshooting.xml [hunk 24]
```diff
@@ -405,7 +405,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
        </listitem>
      </varlistentry>
      <varlistentry>
-       <term>How can I reduce the history output?</term>
+       <term>How can i reduce the history output?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:408]:
```suggestion
       <term>How can I reduce the history output?</term>
```

File: xml/ha_troubleshooting.xml [hunk 25]
```diff
@@ -431,7 +431,7 @@ crm resource cleanup <replaceable>rscid</replaceable> [<replaceable>node</replac
        </listitem>
      </varlistentry>
      <varlistentry>
-       <term>How can I store a <quote>session</quote> for later inspection?</term>
+       <term>How can i store a <quote>session</quote> for later inspection?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:434]:
```suggestion
       <term>How can I store a <quote>session</quote> for later inspection?</term>
```

File: xml/ha_troubleshooting.xml [hunk 26]
```diff
@@ -486,7 +486,7 @@ chmod 640 /etc/hawk/hawk.key /etc/hawk/hawk.pem</screen>
   <variablelist>
    <varlistentry>
 <!-- FATE#311413 -->
-    <term>How can I run commands on all cluster nodes?</term>
+    <term>How can i run commands on all cluster nodes?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:489]:
```suggestion
    <term>How can I run commands on all cluster nodes?</term>
```

File: xml/ha_troubleshooting.xml [hunk 27]
```diff
@@ -576,7 +576,7 @@ Jan 12 16:04:22 &node1; modprobe: FATAL: Module ocfs2_stackglue not found.</scre
    </varlistentry>
    <varlistentry xml:id="vle-ha-crmreport">
 <!-- FATE#310174 -->
-    <term>How can I create a report with an analysis of all my cluster nodes?</term>
+    <term>How can i create a report with an analysis of all my cluster nodes?</term>
```
Comment (taroth21) [inline xml/ha_troubleshooting.xml:579]:
```suggestion
    <term>How can I create a report with an analysis of all my cluster nodes?</term>
```

File: xml/ha_yast_cluster.xml [hunk 28]
```diff
@@ -152,7 +152,7 @@
     </listitem>
    </varlistentry>
    <varlistentry xml:id="vle-ha-rrp">
-    <term>Redundant Ring Protocol (RRP)</term>
+    <term>Redundant ring protocol (RRP)</term>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:155]:
```suggestion
    <term>Redundant Ring Protocol (RRP)</term>
```

File: xml/ha_yast_cluster.xml [hunk 29]
```diff
@@ -200,7 +200,7 @@
  </sect1>
 
  <sect1 xml:id="sec-ha-setup-yast-overview">
-   <title>&yast; Cluster Module</title>
+   <title>&yast; cluster module</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:203]:
```suggestion
   <title>&yast; <guimenu>Cluster</guimenu> module</title>
```

File: xml/ha_yast_cluster.xml [hunk 30]
```diff
@@ -284,7 +284,7 @@
    </para>
 
    <note>
-    <title>Settings in the &yast; Cluster Module</title>
+    <title>Settings in the &yast; cluster module</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:287]:
```suggestion
    <title>Settings in the &yast; <guimenu>Cluster</guimenu> module</title>
```

File: xml/ha_yast_cluster.xml [hunk 31]
```diff
@@ -402,7 +402,7 @@
    </procedure>
 
    <figure>
-    <title>&yast; Cluster&mdash;Multicast Configuration</title>
+    <title>&yast; cluster&mdash;multicast configuration</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:405]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;multicast configuration</title>
```

File: xml/ha_yast_cluster.xml [hunk 32]
```diff
@@ -498,7 +498,7 @@
     </step>
    </procedure>
    <figure>
-    <title>&yast; Cluster&mdash;Unicast Configuration</title>
+    <title>&yast; cluster&mdash;unicast configuration</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:501]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;unicast configuration</title>
```

File: xml/ha_yast_cluster.xml [hunk 33]
```diff
@@ -640,7 +640,7 @@
   </procedure>
 
    <figure>
-    <title>&yast; Cluster&mdash;Security</title>
+    <title>&yast; cluster&mdash;security</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:643]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;security</title>
```

File: xml/ha_yast_cluster.xml [hunk 34]
```diff
@@ -764,7 +764,7 @@
     </step>
    </procedure>
    <figure>
-    <title>&yast; Cluster&mdash;&csync;</title>
+    <title>&yast; cluster&mdash;&csync;</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:767]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;&csync;</title>
```

File: xml/ha_yast_cluster.xml [hunk 35]
```diff
@@ -938,7 +938,7 @@ Finished with 1 errors.</screen>
     </step>
    </procedure>
    <figure xml:id="fig-ha-installation-setup-conntrackd">
-    <title>&yast; Cluster&mdash;<systemitem class="resource">conntrackd</systemitem></title>
+    <title>&yast; cluster&mdash;<systemitem class="resource">conntrackd</systemitem></title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:941]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;<systemitem class="resource">conntrackd</systemitem></title>
```

File: xml/ha_yast_cluster.xml [hunk 36]
```diff
@@ -1000,7 +1000,7 @@ Finished with 1 errors.</screen>
     </step>
    </procedure>
    <figure>
-    <title>&yast; Cluster&mdash;Services</title>
+    <title>&yast; cluster&mdash;services</title>
```
Comment (taroth21) [inline xml/ha_yast_cluster.xml:1003]:
```suggestion
    <title>&yast; <guimenu>Cluster</guimenu>&mdash;services</title>
```

**Review by taroth21 — CHANGES_REQUESTED** [review]
@tomschr : Could you please double-check if my suggested changes make sense? Thank you!

File: xml/ha_concepts.xml [hunk 37]
```diff
@@ -530,7 +530,7 @@
   </para>
 
   <figure>
-   <title>Typical Fibre Channel Cluster Configuration</title>
+   <title>Typical Fibre channel cluster configuration</title>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/ha_concepts.xml:533]: > 'Fibre channel' or Fibre Channel'?

Hmn, ... :thinking: ... I'm not convinced (yet). :slightly_smiling_face:  It's just a a word of two nouns like _glass fibre_, _carbon fibre_ etc. and we wouldn't write these in uppercase too, right? I also would keep the exceptions of our rule to a minimum.

So I would just use in the above case "Typical fibre channel cluster configuration". I might be wrong, maybe @lproven has a better answer.

File: xml/ha_cluster_lvm.xml [hunk 38]
```diff
@@ -54,7 +54,7 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Cluster Logical Volume Manager (&clvm;)</term>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/ha_cluster_lvm.xml:57]: Hmn... probably yes. I think in that case, it justifies uppercase spelling.

File: xml/ha_concepts.xml (hunk 37 above)
[the remark this answers is not in the document]
Comment (sknorr) [reply xml/ha_concepts.xml:533]: Fibre Channel is a trademark afaik, so it needs to be capitalized.

[the remark this answers is not in the document]
Comment (tomschr) [reply xml/ha_concepts.xml:533]: Ahhh, right! Good point. Well, in that case, we need to capitalize it.

**Comment by tomschr** [comment]
@taroth21 Looks good to me. :+1:

**Review by tomschr — APPROVED** [review]

File: xml/ha_concepts.xml (hunk 37 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/ha_concepts.xml]: Corrected to uppercase.

File: xml/ha_cluster_lvm.xml (hunk 38 above)
[the remark this answers is not in the document]
Comment (tomschr) [reply xml/ha_cluster_lvm.xml:57]: We leave it as is (uppercase spelling).

**Pull request merged**